runtime_windows.go

```diff
@@ -0,0 +1,7 @@
+package utils
```

Comment:
Do we need a directive to only build on windows platforms here?

```suggestion
//go:build windows

package utils
```

Reply:
you can use GOOS + GOARCH in file suffixes to get build constraints

https://pkg.go.dev/go/build#hdr-Build_Constraints

(search for `source_windows.go` on that page)